webpack/base.conf.babel.js

```diff
@@ -77,7 +78,17 @@ module.exports = {
       },
       {
         test: /\.s[a|c]ss$/,
-        loader: 'style-loader!css-loader!sass-loader',
+        use: [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              data: `@import "scss/${MODULE_SCHEME_NAME}.variables.scss";`,
```

Comment:
will return later to this
for me:
https://sass-scss.ru/documentation/sassscript/peremennie_po-umolchaniyu.html
https://github.com/webpack-contrib/sass-loader#environment-variables